.github/workflows/ci.yml

```diff
@@ -12,8 +12,7 @@ jobs:
     strategy:
       matrix:
         ruby-version:
-          - 2.7
```

Comment:
?

Reply:
Trying to understand impact

Specifying 3.0 gives version 3.4.1 at the moment. This is causing error.
Do we need to have ruby 2.7 inside this matrix?

Reply:
Let's check which versions are still officially supported and use them

Reply:
`3.0` was previously interpreted as latest 3 which is 3.4, because it should have been released yesterday.
Everything works with string versions